packages/backend/src/query/convertWhereToQueryProviderInput.test.ts

```diff
@@ -0,0 +1,82 @@
+import { describe, expect, test } from 'vitest';
+import { DB, from } from '../tests/generated';
+import { convertWhereToQueryProviderInput } from './convertWhereToQueryProviderInput';
+import { Table } from '@synthql/queries';
+
+describe('convertWhereToQueryProviderInput', () => {
```

Comment:
This test is way too verbose, and is testing way too little cases jim. For example, you're creating an entire query, but not really using it. There's also a lot of code duplication between the test cases

I suggest you use the `test.each` used in other parts of the code. 

In essence you want two different types of cases:
- Case 1: wheres that should convert safely
- Case 2: wheres that should throw an error.